pages/ps/cutoffs/ps1/index.tsx

```diff
@@ -175,12 +176,17 @@ export default function PS1Data() {
                         </div>
 
 
-                        <div className="flex flex-col md:flex-row w-1/2 justify-center">
-                            <Link className="m-3 w-full" href={"/ps/cutoffs/ps1/add"}>
+                        <div className="flex flex-col md:flex-row w-full justify-center">
+                            <Link className="m-3 w-full max-w-xs" href={"/ps/cutoffs/ps1/add?edit=false"}>
                                 <button className="btn btn-outline w-full" tabIndex={-1}>
                                     Add your response?
                                 </button>
                             </Link>
+                            <Link className="m-3 w-full max-w-xs" href={"/ps/cutoffs/ps1/add?edit=true"}>
```

Comment:
Should edit button be visible even if they haven't submitted one? Although I am not sure how many extra requests this will add.